### PR TITLE
✨Source Insightly: Make Connector Compatible with Builder

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
* Make the connector compatible with the builder

## How
* Import schema to manifest
* Update docs, 
* Completely made compatible with the builder

## User Impact
* No impact, chore for maintainability.

## Can this PR be safely reverted and rolled back?
- [x] YES 💚

### CODE REVIEW

**Comment by btkcodedev** [comment]
Removed $parameter macros :white_check_mark: 
Embed schemas :white_check_mark: 

GitHub CI Green, Ready to merge :white_check_mark: 
image

File: airbyte-integrations/connectors/source-insightly/metadata.yaml [hunk 1]
```diff
@@ -4,13 +4,13 @@ data:
     sl: 100
   allowedHosts:
     hosts:
-      - TODO
+      - api.na1.insightly.com
```
Comment (girarda) [inline airbyte-integrations/connectors/source-insightly/metadata.yaml:7]: <3

**Review by girarda — APPROVED** [review]

File: docs/integrations/sources/insightly.md [hunk 2]
```diff
@@ -69,6 +69,7 @@ The connector is restricted by Insightly [requests limitation](https://api.na1.i
 
 | Version | Date       | Pull Request                                             | Subject                                                                         |
 | :------ | :--------- | :------------------------------------------------------- | :------------------------------------------------------------------------------ |
+| 0.2.5   | 2024-05-30 | [38140](https://github.com/airbytehq/airbyte/pull/38140) | Make compatable with builder                                                    |
```
Comment (girarda) [inline docs/integrations/sources/insightly.md:72]: nit: let's update the date

Comment (girarda) [inline docs/integrations/sources/insightly.md:72]:
```suggestion
| 0.2.5   | 2024-05-14 | [38140](https://github.com/airbytehq/airbyte/pull/38140) | Make compatible with builder                                                    |
```

**Comment by btkcodedev** [comment]
Change done

**Pull request merged**